docker-compose.override.yml

```diff
@@ -38,6 +38,17 @@ services:
       - postgres
       - rabbitmq
 
+  yarn:
+    image: node:latest
```

Comment:
Can we pin to an image with the right major version?

Reply:
Yes; changed in [3889dec](https://github.com/atlascope/atlascope/pull/136/commits/3889dec9b0c9a6f633631bd441e20f90ca9b1dcb)

Reply:
Thank you! My approval stands.